api/src/test/java/org/pmiops/workbench/rdr/RdrExportServiceImplTest.java

```diff
@@ -71,7 +72,7 @@
 
   private DbUser dbUserWithEmail;
   private DbUser dbUserWithoutEmail;
-  private DbWorkspace mockWorkspace, mockDeletedWorkspace;
+  private DbWorkspace mockWorkspace, mockDeletedWorkspace, mockCreatorWorkspace;
```

Comment:
nit: Usually I'd use `@Mock` for mocking value objects (and `@MockBean` for beans). There may be a reason not to do so here.